types/storybook__react/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for @storybook/react 3.0
+// Type definitions for @storybook/react 3.4
```

Comment:
should we bump this to 4.0 because before `parameters` were called `options`?

Reply:
Good point – and bumping to 4.0 allows us to add more of the new API to the type definitions. 👍